File: FBOLinx.Web/FBOLinx.Web/Models/FuelReq.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace FBOLinx.Web.Models
{
    public partial class FuelReq
    {
        public int Oid { get; set; }
        public int? CustomerId { get; set; }
        public string Icao { get; set; }
        public int? Fboid { get; set; }
        public int? CustomerAircraftId { get; set; }
        public DateTime? Eta { get; set; }
        public DateTime? Etd { get; set; }
        public string TimeStandard { get; set; }
        public bool? Cancelled { get; set; }
        public double? QuotedVolume { get; set; }
        public double? QuotedPpg { get; set; }
        public string Notes { get; set; }
        public DateTime? DateCreated { get; set; }
        public double? ActualVolume { get; set; }
        public double? ActualPpg { get; set; }
        public string Source { get; set; }
        public int? SourceId { get; set; }
        public string DispatchNotes { get; set; }
        public bool? Archived { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }

        [ForeignKey("CustomerId")]
        [InverseProperty("FuelReqs")]
        public Customers Customer { get; set; }

        [ForeignKey("CustomerAircraftId")]
        [InverseProperty("FuelReqs")]
        public CustomerAircrafts CustomerAircraft { get; set; }

        [ForeignKey("Fboid")]
        [InverseProperty("FuelReqs")]
        public Fbos Fbo { get; set; }

        [InverseProperty("FuelReq")]
        public FuelReqPricingTemplate FuelReqPricingTemplate { get; set; }
    }
}

File: FBOLinx.Web/FBOLinx.Web/ClientApp/src/app/models/fuelreq.ts
//Create a FuelReq type that reflects the FuelReqDto

import { Customer } from "./customer"
import { CustomerAircraft } from "./customer-aircraft"
import { ServiceOrderItem } from "./service-order-item"

export type FuelReq = {
    oid: number;
    customerId: number;
    fboId: number;
    customerInfoByGroupId: number;
    arrivalDateTimeLocal?: Date;
    departureDateTimeLocal?: Date;
    eta?: Date;
    etd?: Date;
    dateCreated?: Date;
    icao: string;
    customerAircraftId: number;
    timeStandard: string;
    cancelled?: boolean;
    quotedVolume?: number;
    quotedPpg?: number;
    notes: string;
    actualVolume?: number;
    actualPpg?: number;
    source: string;
    sourceId?: number;
    dispatchNotes: string;
    archived?: boolean;
    email: string;
    phoneNumber: string;
    fuelOn: string;
    customerName: string;
    customerNotes: string;
    paymentMethod: string;
    timeZone: string;
    isConfirmed?: boolean;
    tailNumber: string;
    fboName: string;
    pricingTemplateName: string;

    serviceOrderItems: ServiceOrderItem[]
    customer: Customer;
    customerAircraft: CustomerAircraft;
    //fbo: 
    //FuelReqPricingTemplate
}
